sdk/servicebus/service-bus/src/connectionContext.ts

```diff
@@ -66,7 +66,7 @@ export namespace ConnectionContext {
 
     // Define listeners to be added to the connection object for
     // "connection_open" and "connection_error" events.
-    const onConnectionOpen: OnAmqpEvent = (context: EventContext) => {
+    const onConnectionOpen: OnAmqpEvent = () => {
```

Comment:
Doesn't this break the type signature? You might just need to change the parameter name to `_` if it's unused.

Reply:
I did this in a bunch of places, then I think i should do it everywhere

Reply:
Changing the parameter name doesn't really help, should i just turn off that rule instead? 
`" '_' is defined but never used." @typescript-eslint/no-unused-vars`

Got the above error.

Reply:
If the build doesnt fail, then I am fine with this change

Reply:
Yes build doesn't fail